### PR TITLE
feat(connectWallet): refactor some props

### PR DESCRIPTION
**Pull request by zctocm**
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/rigel/38)

### CODE REVIEW

File: src/pages/App.js [hunk 1]
```diff
@@ -3,36 +3,36 @@ import {BrowserRouter as Router, Route, Switch} from 'react-router-dom'
 import Shuttle from '../pages/Shuttle'
 import History from '../pages/History'
 import TokenList from '../pages/Shuttle/TokenList'
-// import Web3ReactManager from '../pages/components/Web3ReactManager'
+import Web3ReactManager from '../pages/components/Web3ReactManager'
```
Comment (angelia-yuqi-personal) [inline src/pages/App.js:6]: `Web3ReactManager` can be exported from components/index

File: src/pages/Example.js [hunk 2]
```diff
@@ -151,6 +152,7 @@ function Example() {
         chain="eth"
         open={false}
       ></ConnectWalletModal>
+      <AccountStatus chain="eth" size="large"></AccountStatus>
```
Comment (angelia-yuqi-personal) [inline src/pages/Example.js:155]: when component has no child, it can end with `/` instead of close tag with simple code

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

File: src/pages/App.js (hunk 1 above)
Comment (zctocm) [reply src/pages/App.js]: done

File: src/pages/Example.js (hunk 2 above)
Comment (zctocm) [reply src/pages/Example.js]: done

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**